examples/pom.xml

```diff
@@ -11,23 +11,64 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <!-- change to the version you want to use. -->
     <opencensus.version>0.12.2</opencensus.version><!-- LATEST_OPENCENSUS_RELEASE_VERSION -->
+    <grpc.version>1.9.0</grpc.version><!-- CURRENT_GRPC_VERSION -->
```

Comment:
The same.

Reply:
Same reason, have to use v1.9.0 until we update the example to use opencensus v0.13.0.